Source/Scene/Camera.js

```diff
@@ -3727,48 +3821,30 @@ Camera.prototype.computeViewRectangle = function (ellipsoid, result) {
   var width = canvas.clientWidth;
   var height = canvas.clientHeight;
 
-  var successfulPickCount = 0;
+  var unsuccessfulPicks = [];
 
-  var computedHorizonQuad = computeHorizonQuad(this, ellipsoid);
-
-  successfulPickCount += addToResult(
-    0,
-    0,
-    0,
-    this,
-    ellipsoid,
-    computedHorizonQuad
-  );
-  successfulPickCount += addToResult(
-    0,
-    height,
-    1,
-    this,
-    ellipsoid,
-    computedHorizonQuad
-  );
-  successfulPickCount += addToResult(
-    width,
-    height,
-    2,
-    this,
-    ellipsoid,
-    computedHorizonQuad
-  );
-  successfulPickCount += addToResult(
-    width,
-    0,
-    3,
-    this,
-    ellipsoid,
-    computedHorizonQuad
-  );
+  if (!addToResult(0, 0, UL, this, ellipsoid)) {
+    unsuccessfulPicks.push(UL);
+  }
+  if (!addToResult(0, height, LL, this, ellipsoid)) {
+    unsuccessfulPicks.push(LL);
+  }
+  if (!addToResult(width, height, LR, this, ellipsoid)) {
+    unsuccessfulPicks.push(LR);
+  }
+  if (!addToResult(width, 0, UR, this, ellipsoid)) {
+    unsuccessfulPicks.push(UR);
+  }
 
-  if (successfulPickCount < 2) {
+  if (unsuccessfulPicks.length > 2) {
     // If we have space non-globe in 3 or 4 corners then return the whole globe
     return Rectangle.MAX_VALUE;
   }
 
+  if (unsuccessfulPicks.length > 0) {
+    adjustViewForHorizon(this, unsuccessfulPicks, ellipsoid);
```

Comment:
There should be an entry in `CHANGES.md` under the "Fixes" section for this.

Reply:
Added